### PR TITLE
[13.0][MIG] shopinvader_product_template_multi_link

### PR DESCRIPTION
**Pull request by simahawk**
Replaces and improves https://github.com/shopinvader/odoo-shopinvader/pull/513

### CODE REVIEW

File: shopinvader_product_template_multi_link/tests/test_product.py [hunk 1]
```diff
@@ -6,12 +6,49 @@
 
 
 class ProductCase(ProductCommonCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```
Comment (Tonow-c2c) [inline shopinvader_product_template_multi_link/tests/test_product.py:11]: Maybe 

```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

**Review by Tonow-c2c — APPROVED** [review]
Just code review

File: shopinvader_product_template_multi_link/tests/test_product.py (hunk 1 above)
Comment (simahawk) [reply shopinvader_product_template_multi_link/tests/test_product.py:11]: it's not needed because it's done on the base mixin for tests :wink:

Comment (Tonow-c2c) [reply shopinvader_product_template_multi_link/tests/test_product.py:11]: thanks :+1:

**Review by lmignon — APPROVED** [review]
LGTM Code review (last commit)

**Pull request merged**